index.js

```diff
@@ -313,22 +415,31 @@ async function createVcExamples() {
       });
     }
 
-    // set up the unsigned button
-    addTab('unsigned', 'Verifiable Credential', () => example.outerHTML);
-
-    if(tabTypes.indexOf(suiteEd25519Signature2020.type) > -1) {
-      await addProofTab(suiteEd25519Signature2020);
+    function hasTab(identifier) {
+      return tabTypes.indexOf(identifier) > -1;
     }
-    if(tabTypes.indexOf(suiteEd25519Multikey.cryptosuite) > -1) {
-      await addProofTab(suiteEd25519Multikey);
+
+    // set up the unsigned button
+    addTab(
+      'unsigned',
+      'Unsecured credential',
+      'Credential',
+      () => example.outerHTML
+    );
+
+    for(const {proof, key, label} of exampleProofs) {
+      if(hasTab(proof.cryptosuite)) {
+        await addProofTab(proof, label, key);
+      }
     }
-    if(tabTypes.indexOf(suiteEcdsaMultiKey.cryptosuite) > -1) {
-      await addProofTab(suiteEcdsaMultiKey);
+
+    if(hasTab(suiteEd25519Signature2020.type)) {
+      await addProofTab(suiteEd25519Signature2020, 'Ed25519Signature2020');
```

Comment:
Note this is no longer in the original order, but it doesnt seem that matters. Also note current build does not show this tab. I assume that is intentional.

Reply:
@tminard it's old, so we took it out of the defaults back in https://github.com/w3c/respec-vc/commit/91f42f7ef223d83b4995b3ba706471a9666e4587

It's still documented as available via `data-vc-tabs` though.